exporters/trace/stackdriver/src/main/java/io/opencensus/exporter/trace/stackdriver/StackdriverV2ExporterHandler.java

```diff
@@ -260,6 +272,35 @@ public Void apply(Long longValue) {
     return attributeValueBuilder.build();
   }
 
+  private static Link.Type toLinkTypeProto(io.opencensus.trace.Link.Type type) {
+    if (type == io.opencensus.trace.Link.Type.PARENT_LINKED_SPAN) {
+      return Link.Type.PARENT_LINKED_SPAN;
+    } else {
+      return Link.Type.CHILD_LINKED_SPAN;
+    }
+  }
+
+  private static Link toLinkProto(io.opencensus.trace.Link link) {
+    checkNotNull(link);
+    return Link.newBuilder()
+        .setTraceId(encodeTraceId(link.getTraceId()))
+        .setSpanId(encodeSpanId(link.getSpanId()))
+        .setType(toLinkTypeProto(link.getType()))
+        .setAttributes(toAttributesBuilderProto(link.getAttributes(), 0))
```

Comment:
Good catch. Short answer, yes. According to the [API document]:(https://cloud.google.com/trace/docs/reference/v2/rpc/google.devtools.cloudtrace.v2#google.devtools.cloudtrace.v2.Span.Attributes)
>Attributes can be discarded because their keys are too long or because there are too many attributes.

I don't think we have such logic for now. I'll add that later.

Reply:
Should there be a TODO or issue, so that we don't forget to also update this line?